### DISCUSSION TITLE
Propose pdm as an alternative package manager on repl.it

### DISCUSSION THREAD

**Discussion by pohlt** [pdm-project/pdm#317] [General]
I'm using [repl.it](https://repl.it/) from time to time and they are using poetry for package management.

Since I like pdm, I have opened the proposal to add pdm an an alternative package manager. You can upvote my proposal [here](https://replit.com/talk/ask/Support-PDM-as-package-manager-as-an-alternative-to-poetry/129708). As a side effect, this would draw some more attention to pdm. ;-)

**Comment by pohlt** [comment]
@frostming Hmm, seems like we could [bring PDM to repl.it](https://blog.replit.com/upm).

**Reply by frostming** [reply]
Sounds good, can you help submit a UPM for pdm? If you need any assistance just let me know
